statistics/handle.go

```diff
@@ -156,3 +156,32 @@ func (h *Handle) UpdateTableStats(tables []*Table, deletedIDs []int64) {
 	}
 	h.statsCache.Store(newCache)
 }
+
+// LoadNeededHistograms will load histograms for those needed columns.
+func (h *Handle) LoadNeededHistograms() error {
+	histogramNeededColumns.m.Lock()
```

Comment:
I think we can implement a `keys` function which returns a slice to extract all the keys. And then implement a remove function to remove just one key.